com.regnosys.rosetta/src/com/regnosys/rosetta/generator/java/util/ModelNamespaceUtil.xtend

```diff
@@ -0,0 +1,20 @@
+package com.regnosys.rosetta.generator.java.util
+
+import com.google.common.collect.LinkedHashMultimap
+import com.regnosys.rosetta.rosetta.RosettaModel
+import java.util.List
+
+class ModelNamespaceUtil {
```

Comment:
Make it public if you want to use this anywhere esle.

Reply:
public modifier is not needed in xtend files apparently :)